compiler/testData/codegen/boxInline/smap/anonymous/lambda.kt

```diff
@@ -1,5 +1,4 @@
 // FILE: 1.kt
-// IGNORE_BACKEND: JVM_IR
 package builders
```

Comment:
@sfs Please just add new line here and in next file too

Reply:
Done